src/main/java/org/sonatype/repository/conan/internal/hosted/ConanHostedFacet.java

```diff
@@ -213,12 +213,15 @@ private Content saveAsset(final StorageTx tx,
 
   /**
    * Services the download_urls endpoint for root and package data
+   * @param path path as GAV
```

Comment:
Trivial: path path is not necessary

Reply:
Maybe rename this to gavPath?

Reply:
I would like a comment to explain that artifacts are stored with one path and fetched with another + the reason why (other endpoints use GAV). Weird that the Conan developers decided to be inconsistent here - if you've created an issue with conan IO maybe link it here.